docs/source/api_references.rst

```diff
@@ -33,6 +33,30 @@ Core API
     hooks
     lightning
 
+Strategy API
+------------
+
+.. currentmodule:: pytorch_lightning.strategies
+
+.. autosummary::
+    :toctree: api
+    :nosignatures:
+    :template: classtemplate.rst
+
+    Strategy
+    SingleDeviceStrategy
+    ParallelStrategy
+    DataParallelStrategy
+    DDPStrategy
+    DDP2Strategy
+    DDPShardedStrategy
+    DDPSpawnShardedStrategy
+    DDPSpawnStrategy
+    DeepSpeedStrategy
+    HorovodStrategy
+    SingleTPUStrategy
+    TPUSpawnStrategy
```

Comment:
`DDPFullyShardedStrategy` is missing here?

Reply:
probably got forgotten at the time these plugins were introduced, including IPUPlugin :)

Reply:
there is an issue compiling docs if we include. needs to be looked into separately.